src/components/LandingPage/ProductFeatureIcons/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import analyticsImg from '../images/ProductFeatureIcons/analytics.svg'
+import heatmapsImg from '../images/ProductFeatureIcons/heatmaps.svg'
+import sessionReplayImg from '../images/ProductFeatureIcons/session-replay.svg'
+import featureFlagsImg from '../images/ProductFeatureIcons/feature-flags.svg'
+import userFeedbackImg from '../images/ProductFeatureIcons/user-feedback.svg'
+import revenueTrackingImg from '../images/ProductFeatureIcons/revenue-tracking.svg'
+
+interface ProductIconData {
+    label: string
+    icon: any
```

Comment:
[non-blocking] the `src` attribute of an `img` tag can only take a string, so might as well make this `string` too